Automattic/wp-calypso
# Domains: Show auto renewal date for domains that have auto renew turned on

#### Changes proposed in this Pull Request

This PR updates the domains status card to show the renewal date instead of the expiration date when auto renew is on.

<img width="730" alt="Screenshot 2020-06-16 at 8 38 11 PM" src="https://user-images.githubusercontent.com/13062352/84815792-d6a18c00-b013-11ea-8fc9-b266a49b8feb.png">


#### Testing instructions

* For a domain you own, enable auto renew
* Verify that it shows the renewal date instead of the expiration date, like in the screenshot above
* Check that the expiration date is shown instead if you disable auto-renew
